docs/products/metrics/concepts/metrics-overview.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Aiven for Metrics® overview
+---
+
+Aiven for Metrics®, powered by Thanos, simplifies managing and analyzing large volumes of metrics data for businesses of all sizes. It offers a scalable, reliable, and efficient service suitable for businesses of all sizes. This service simplifies the management of large-scale metrics systems, allowing organizations to focus on deriving insights from their data.
+
+## Key components
+
+Aiven for Metrics incorporates several core Thanos components:
+
+- **Thanos Metrics Query**: Allows users to query and visualize metrics from real-time
+  and historical data sources, aggregating data from various sources for a unified view.
+- **Thanos Receiver**: Handles metrics ingestion into the system, acting as a receiver
+  for Prometheus remote write requests to enable real-time metrics collection.
+- **Thanos Metric Store**: Interfaces with object storage to provide access to historical
+  data, ensuring scalable and reliable long-term storage.
+- **Thanos Metrics Compact**: Enhances storage usage and query efficiency by compacting
+  and downsampling data stored in object storage, improving performance and
+  reducing costs.
+- **Object Storage**: Serves as the primary storage solution, offering a durable and
+  scalable way to store extensive amounts of metric data.
+
+## Unified cluster architecture
+
+Aiven for Metrics combines these components into a cohesive cluster architecture.
+This setup ensures a seamless data flow from ingestion (via Thanos Metrics Receive) to
+long-term storage (in object storage through Thanos Metrics Store) and efficient querying
```

Comment:
The Store actually only query data from object storage. 
Data is written to object storage by receivers (when TSDB block is filled, every 2 hours). 
Query Frontend forwards requests to Query, Query asks Receivers and Store depending on the data range requested, deduplicates data (since multiple Receivers can have the same samples) and answers to Query Frontend which same results in cache before sending to the client.

Reply:
Update this section, please take a look.